app/src/views/private/components/header-bar.vue

```diff
@@ -16,7 +16,7 @@
 			<div class="title">
 				<slot name="title">
 					<slot name="title:prepend" />
-					<h1 class="type-title">{{ title }}</h1>
+					<h1 class="type-title" :title="title">{{ title }}</h1>
```

Comment:
Presumably this is to resolve text that are "cut off" due to not enough space, coincidentally I've been wondering about this recently! 

I think we should probably use the `<v-text-overflow :text="title" placement="bottom">` from a design consistency perspective as we use that throughout the app. (understandably I do also assume the title attribute would have technically be better in terms of a11y if I'm not mistaken...)

Reply:
Yesss 🚀